Sources/PackageModel/Package.swift

```diff
@@ -111,12 +122,6 @@ extension Package.Error: Equatable {
         switch (lhs, rhs) {
         case let (.noManifest(lhs), .noManifest(rhs)):
             return lhs.baseURL == rhs.baseURL && lhs.version == rhs.version
-        case (.noManifest, _):
```

Comment:
i removed this because this was showing warning: case is already handled by previous patterns; consider removing it